### PR TITLE
Added explanations about migrations

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | #2641
| License       | MIT

WIP. I want to know if I'm on the good way. 

- [x] list all SQL code for shared hosting

### CODE REVIEW

**Comment by nicosomb** [comment]
:fr: ✅ 
:de: need @jlnostr and @Strubbl help ;-)

**Comment by nicosomb** [comment]
thank you @jlnostr! 

Now I have to write SQL code for shared hosting users.

**Comment by nicosomb** [comment]
@j0k3r with this [migration](https://github.com/wallabag/wallabag/blob/2.2/app/DoctrineMigrations/Version20161001072726.php), I can't write pure SQL for the documentation. @tcitworld suggested me yesterday to write a PHP file for our shared hosted users. What do you think about that?

**Comment by j0k3r** [comment]
Could be an alternative ...

**Comment by nicosomb** [comment]
As I discussed on gitter to @tcitworld, if we create a new PHP file, we need to add in a new bundle (to access to the database parameters). 
**But**, to run a command in the bundle, we need to have a database up-to-date. 

Or a single PHP file which will include parameters.yml. 

Or all the queries in pure SQL.

**Comment by tcitworld** [comment]
Rebase me ❤️

**Comment by nicosomb** [comment]
I rebased you. 

And I added documentation (hardcoded SQL).

File: scripts/update.sh [hunk 1]
```diff
@@ -16,4 +16,5 @@ git fetch --tags
 TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
 git checkout $TAG --force
 SYMFONY_ENV=$ENV $COMPOSER_COMMAND install --no-dev -o --prefer-dist
+php bin/console doctrine:migrations:migrate --env=$ENV
```
Comment (j0k3r) [inline scripts/update.sh]: You should add `--no-interaction` otherwise it'll ask the user

**Pull request merged**